### ISSUE TITLE
Can not embed the video from YouTube or Microsoft Stream

### ISSUE THREAD

**Issue by jvennur** [pnp/custom-learning-office-365#225]
>**PROVISIONING NOTE**: If you are having issues with the provisioning service while trying to provision M365 learning pathways please do not post that issue to this repo as we cannot help you here. Please post all questions and issues for the provisioning service to [https://github.com/SharePoint/sp-provisioning-service/issues](https://github.com/SharePoint/sp-provisioning-service/issues).

**Could not able to embed the videos from youtube and also from the Microsoft stream**

**To Reproduce**
1. Go to '...'  Assets from for Teams or any of the workloads
2. Click on '....' add an asset with the link to the Video
3. Scroll down to '....' Try to open the page 

**Expected behavior**

**Screenshots**

**Learning Pathways version number:**
Please share the version number of learning pathways in your tenant app catalog.

**Additional context**
Added recording with the issue Repro  , please rename the file .zip and review the case.

**Comment by juliemturner** [comment]
Can you provide some clarity? Are you trying to add a custom asset to a playlist in the Admin web part? And if that's the case are you saying you're trying to add an asset that's a link to a video? 

You didn't provide the error or a screenshot so it's a bit hard to follow what the problem is.  If you could pelase provide more information I'll try and assist further.

**Comment by juliemturner** [comment]
Ok, I think you've duplicated this with a new issue.  I"m going to change the title of this issue to match #226. 

And then to answer you, you cannot put a link to a YouTube video or Stream video directly in the asset url. You must create a SharePoint page and then embed the Stream or YouTube video in that page.  YOu can create the page using the asset UI or you can paste the link to the SharePoint page directly in like you tried to do with the video.

**Comment by jvennur** [comment]
Hi Julie ,

Yes , I am trying to add a New Asset to the play list  with a link to video  , let me know if you would like to repro we can do over the Teams screen share.

Regards,
Raana Krishna Vennu

[cid:image001.png@01D5FE03.27E08440]

**Comment by juliemturner** [comment]
Just verifying that I've answered this question, I think you've responded after I did so. Long story short you cannot paste the youtube video url directly into the asset list you must embed it in a SharePoint page first.

> Ok, I think you've duplicated this with a new issue. I"m going to change the title of this issue to match #226.
> 
> And then to answer you, you cannot put a link to a YouTube video or Stream video directly in the asset url. You must create a SharePoint page and then embed the Stream or YouTube video in that page. YOu can create the page using the asset UI or you can paste the link to the SharePoint page directly in like you tried to do with the video.
